cfgov/unprocessed/apps/find-a-housing-counselor/css/hud.less

```diff
@@ -15,7 +15,13 @@
 }
 
 .hud_hca_api_results_actions {
-    float: right;
+    .respond-to-min( @bp-xs-max, {
+        & {
+            float: right;
+            // Provide a gap between result header links and result text.
+            margin-left: 20px;
```

Comment:
If I want to nitpick, `margin` should come before `float`, but I don't care that much.

Reply:
Sure, updated!